testing_and_setup/compass/ocean/global_ocean/config_files/config_e3sm_coupling.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<config case="e3sm_coupling">
+
+	<add_link source="../initial_state/initial_state.nc" dest="init.nc"/>
+	<add_link source="../initial_state/graph.info" dest="graph.info"/>
+    <add_link source_path="script_configuration_dir" source="scripts/create_E3SM_coupling_files.py" dest="create_E3SM_coupling_files.py"/>
+    <add_link source_path="script_configuration_dir" source="scripts/create_E3SM_coupling_files.ini" dest="create_E3SM_coupling_files.ini"/>
```

Comment:
I don't know if this is possible with COMPASS as currently written but this should probably be a copy of the file rather than a link so it could be altered by the user without affecting other test cases.

Reply:
I thought links would be better so that we are not copying a number of large files, for time and file space. I could put a `readme` file in there with the cp and tar commands to get full copies.

Reply:
I don't think space is a concern for the `.ini` file, which is the only one I was referring to.

Reply:
Good idea. I did not see a copy_file or similar capability in setup_testcase.py, but I see how to do it.  That would be a separate PR. I put it on a to-do list. Leaving it as a link for now.